tools/shared.py

```diff
@@ -432,6 +432,7 @@ def check_fastcomp():
 EXPECTED_NODE_VERSION = (0,8,0)
 
 def check_node_version():
+  jsrun.check_engine(NODE_JS)
```

Comment:
Well, actually this location I could go either way, since we check in the code below anyhow.